python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -334,7 +350,16 @@ def _setParent(self, parentObj):
 
     def getCachedData(self, index):
 
-        return
+        raw_self = stripProxy(self)
+        if index > len(raw_self) or index < 0:
+            return None
+
+        if index in self._subjobIndexData.keys():
+            return self._subjobIndexData[index]
+        else:
+            return raw_self._registry.getIndexCache( stripProxy(raw_self.__getitem__(i)) )
+
+        return None
```

Comment:
This last bit should be:
```python
if index in self._subjobIndexData.keys():
    return self._subjobIndexData[index]

return raw_self._registry.getIndexCache( stripProxy(raw_self.__getitem__(i)) )
```
and I guess that `i` on that last line should be `index`?

Also, is there a reason for using `__getitem__` rather than `raw_self[index]`?